settings.mk

```diff
@@ -229,6 +229,9 @@ else
 	ifeq ($(EXIT_FAILURE), true)
 		EXIT_CONDITION+=|| $(HAS_FAILURE)
 	endif
+	ifeq ($(EXIT_SUCCESS), true)
+		EXIT_CONDITION+=|| [ $$success -ne 0 ]
```

Comment:
Isn't this adding a success condition?  I'm confused :-)

Reply:
This is part of the repeat feature: https://github.com/adoptium/TKG/issues/352. When we repeat the tests, we would like to have the option EXIT_SUCCESS to stop repeating when encountering a success in last iteration. This is useful for triaging purpose.

Reply:
@karianna - there are cases where we want to prove a test can pass, so run for X iterations or until it passes (exit on first instance of it passing).  There are cases where we want to prove a test will fail, so run for X iterations or until if fails (exit on first instance of it failing), and finally cases where we want to run for X iterations and gather the pass/fail rate for the entire run (do not exit until X iterations are complete).